plugins/palo_alto_cortex_xdr/icon_palo_alto_cortex_xdr/tasks/monitor_alerts/task.py

```diff
@@ -213,6 +213,13 @@ def _parse_custom_config(
                 self.logger.info(f"Saved time {saved_time_str} exceeds cut off, moving to {comparison_str}.")
                 state[LAST_ALERT_TIME] = comparison_unix
 
+                # Reset the offsets when changing search timer
+                state[LAST_SEARCH_FROM] = 0
+                state[LAST_SEARCH_TO] = custom_config.get("alert_limit", ALERT_LIMIT)
```

Comment:
there's logic for parsing this alert limit to make sure it doesn't exceed ALERT_LIMIT, don't think we want to access the limit value directly in here? 

And this looks like on the first run after the 'reset' it would then query to 2000? As we pull the value held in the [state + the next batch](https://github.com/rapid7/insightconnect-plugins/blob/2fcfff9d6eb9beedeb0e48163981635cd9d1e75e/plugins/palo_alto_cortex_xdr/icon_palo_alto_cortex_xdr/tasks/monitor_alerts/task.py#L92-L93) using the liit value?